src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1047,4 +1047,16 @@
   <data name="SNI_PN9" xml:space="preserve">
     <value>SQL Network Interfaces</value>
   </data>
+  <data name="AZURESQL_GenericEndpoint" xml:space="preserve">
+    <value>.database.windows.net</value>
+  </data>
+  <data name="AZURESQL_GermanEndpoint" xml:space="preserve">
+    <value>.database.cloudapi.de</value>
+  </data>
+  <data name="AZURESQL_UsGovEndpoint" xml:space="preserve">
+    <value>.database.usgovcloudapi.net</value>
+  </data>
+  <data name="AZURESQL_ChinaEndpoint" xml:space="preserve">
+    <value>.database.chinacloudapi.cn</value>
+  </data>
```

Comment:
@justinvp No they are not meant to be Localized. We have been adding such strings to resource strings so that any modifications are limited to the Resource file. We have carried this practice over from the ADO.Net development. In ADO.Net we mark these resources as non-localizable so that they are not translated. 
I don't expect these end points to change, however as a team practice we are following this convention.